taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path

```

Comment:
Attach screenshots of pages

Reply:
![image](https://user-images.githubusercontent.com/65457374/219122413-c5246824-0b82-404f-a487-87f7def8d934.png)